primitives/storage/src/lib.rs

```diff
@@ -165,8 +165,15 @@ pub mod well_known_keys {
 	/// Number of wasm linear memory pages required for execution of the runtime.
 	///
 	/// The type of this value is encoded `u64`.
+	///
+	/// Note that this value is used for consensus-related runtime operations, such as block import.
 	pub const HEAP_PAGES: &'static [u8] = b":heappages";
 
+	/// Number of wasm linear memory pages given to offchain workers.
+	///
+	/// The type of this value is encoded `u64`.
```

Comment:
```suggestion
	/// The type of this value is encoded `u64`.
	///
	/// Override `HEAP_PAGES` for offchain calls.
```
Not very good comment suggestion (please don't use it as is :), just underlining that in my opinion offchain should be the special case, and consensus the default, so additional doc in offchain.

Reply:
updated the comment a bit.